website/data/docs-nav-data.json

```diff
@@ -805,6 +805,10 @@
       {
         "title": "FAQ",
         "path": "troubleshoot/faq"
+      },
+      {
+        "title": "Service-to-Service Troubleshooting",
+        "path": "troubleshoot/troubleshoot-services"
```

Comment:
can we move this to the top of the trouble shooting section? i think it'lll look better 

<img width="331" alt="image" src="https://user-images.githubusercontent.com/7556325/220872938-588fe91a-5992-4c3b-8353-7782ad762aa8.png">